dapps/marketplace/src/pages/listing/_ListingSold.js

```diff
@@ -2,21 +2,65 @@ import React from 'react'
 import { fbt } from 'fbt-runtime'
 
 import Link from 'components/Link'
+import Price from 'components/Price'
 
-const Sold = () => (
-  <div className="listing-buy pending">
-    <fbt desc="listingSold.text">
-      <div>This listing is</div>
-      <div>Sold</div>
-      <div>
-        This listing is sold out. Try visiting the listings page and searching
-        for something similar.
+const Sold = ({ listing, isSingleUnit }) => (
+  <div className="listing-buy">
+    <div className="price">
+      <Price listing={listing} descriptor />
+    </div>
+    <div className="status">
+      <div className="status-title sold-out">
+        {isSingleUnit ? fbt('Sold', 'Sold') : fbt('Sold Out', 'SoldOut')}
       </div>
-    </fbt>
-    <Link to="/listings">
-      <fbt desc="viewListings">View Listings</fbt>
-    </Link>
+      <div className="status-text">
+        {!isSingleUnit
+          ? fbt(
+              'This listing is no longer available.',
+              'MultiUnitListing.soldOut'
+            )
+          : fbt(
+              'Another buyer has purchased this listing.',
+              'UnitListing.soldOut'
+            )}
+      </div>
+      <Link className="listing-action-link" to="/listings">
+        <fbt desc="viewAllListings">View all listings</fbt>
+      </Link>
+    </div>
   </div>
 )
 
 export default Sold
+
+require('react-styl')(`
+  .listing-buy
+    .status
+      text-align: center
+      padding: 1.25rem
+      background-color: #f3f7f9
```

Comment:
Are there variables available for these colors? Might need to go through and look at CSS vars again soon

Reply:
It was not there. Can add it if you are fine with it.

Reply:
Perhaps we should review all the CSS vars to ensure consistency. Lets come back to it later